src/playbacks/html5_video/html5_video.js

```diff
@@ -64,7 +64,7 @@ export default class HTML5Video extends Playback {
       'playing': '_onPlaying',
       'progress': '_onProgress',
       'seeked': '_handleBufferingEvents',
-      'seeking': '_handleBufferingEvents',
+      'seeking': '_onSeeked',
```

Comment:
`The seeking event is fired when a seek operation **began**.`

https://developer.mozilla.org/en-US/docs/Web/Events/seeking

Aren't we looking for the event after this?

We should tied this event to `seeked` https://developer.mozilla.org/en-US/docs/Web/Events/seeked